llvm/test/Transforms/LoopVectorize/AArch64/sleef-calls.ll

```diff
@@ -1,4 +1,4 @@
-; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --filter "(_)|(cos|expf?\(|exp2|exp10|fmod|gamma|log|pow|sin|sqrt|tan)|(ret)" --version 2
```

Comment:
In some tests we explicitly check for exp variants (ie, with `expf?\(|exp2|exp10`), and this change groups them together. I don't mind which one to use but it's better to be consistent about it in all relevant tests., eg:
sleef-intrinsic-calls-aarch64.ll
sleef-calls-aarch64.ll
armpl-intrinsics.ll

Reply:
done